tests/FaultyGraphSegmentationTest.java

```diff
@@ -4,7 +4,6 @@
 import sbr.*;
 
 public class FaultyGraphSegmentationTest {
-    @Test
     public void faulty3x3Graph() throws Exception {
```

Comment:
At this file, each test need to have the annotation @Test. Make sure to put it before each test and to keep the first letter capital.